### PR TITLE
Clearer instructions for viewing allure report

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

[description]

medic/cht-core#[number]

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -18,21 +18,28 @@ They live in the `tests` directories of each app. Run them with grunt: `grunt un
...
 
-### WebdriverIO Local Run
+### WebdriverIO
+
+#### Run locally
+
+1. Run `npm ci`
+2. Run `grunt`
+3. Run `npm run wdio-local`
+
```
Comment (ngaruko) [inline TESTING.md:37]: Not a major, but there is a step here about Viewing the report on local run with `npx allure open`

**Review by ngaruko — CHANGES_REQUESTED** [review]
Looks good. Looks like we missed the small step on viewing report on local... `npx allure open`

File: TESTING.md (hunk 1 above)
Comment (garethbowen) [reply TESTING.md:38]: Good catch - fixed.

**Review by ngaruko — APPROVED** [review]
Great...

**Pull request merged**